modules/entity_meta_relation_node/config/schema/entity_meta.entity_type.schema.yml

```diff
@@ -0,0 +1,12 @@
+entity_meta.entity_meta_type.*:
+  type: config_entity
+  label: 'Entity meta type config'
+  mapping:
+    id:
+      type: string
+      label: 'ID'
+    label:
+      type: label
+      label: 'Label'
+    uuid:
```

Comment:
Not needed to define the `uuid` again. it's defined by the parent `config_entity` you are extending (above `type: config_entity`). You can check out many existing core data types in `core.data_types.schema.yml`.